packages/testing-utils/src/index.js

```diff
@@ -0,0 +1,36 @@
+import { TASK } from '@redux-saga/symbols'
+
+export const cloneableGenerator = generatorFunc => (...args) => {
```

Comment:
we continue keeping tests in the core package?

Reply:
no, need to move them too, thanks for reminding about it